ts/lib/i18n/utils.ts

```diff
@@ -48,6 +48,10 @@ export function toLocaleString(
     return date.toLocaleDateString(langs, options);
 }
 
+export function toLocaleNumber(n: number): string {
```

Comment:
I recommend we call this "localizedNumber", and rename "toLocaleString" to "localizedDate"